tests/test_contrib_cern.py

```diff
@@ -138,6 +139,16 @@ def test_account_setup(app, example_cern, models_fixture):
 
         login_user(user)
         assert len(g.identity.provides) == 7
+
```

Comment:
Can you please add something like (pseudo-code):
```
login()
assert type(g.identity) == Identity
assert g.identity.provides == Need(method='role', value='cern-user'), Need(method='system_role', value='authenticated_user'), Need(method='id', value='<email>@cern.ch'), Need(method='system_role', value='any_user'), Need(method='role', value='admin')

logout()
assert type(g.identity) == AnonymousIdentity
assert g.identity.provides == Need(method='system_role', value='any_user')

login()
...test again...
```
as in the GitHub issue.
Same for the other tests.

Reply:
To follow our morning catch up, when we log out the `AnonymousIdentity` provides are `UserNeed({userId})`, instead of the generic `RoleNeed('any_user')`. I have left a [note](https://github.com/topless/invenio-oauthclient/blob/05cd1be4f99b499241b3a4bd3191b5cf22763237/tests/test_contrib_cern_rest.py#L145) in one of the tests.